draft-ietf-core-dns-over-coap.md

```diff
@@ -511,6 +511,11 @@ harden against injecting spoofed responses.
 Consequently, it is of little concern to leverage the benefits of CoAP caching by setting the ID to
 0.
 
+A user of DoC must be aware that the DoC server, unless the security context is with yet another
+party, may communicate unencrypted with the upstream DNS infrastructure, e.g., using DNS over UDP.
```

Comment:
```suggestion
A user of DoC must be aware that the DoC server
may communicate unencrypted with the upstream DNS infrastructure, e.g., using DNS over UDP.
```

If the security context is with yet another party, that "DoC server" would just be a reverse proxy (which is likely not even aware that DoC is going on) and not a DoC server, and that other party would be the DoC server.